astropy/stats/sigma_clipping.py

```diff
@@ -605,11 +609,12 @@ def __call__(
             If ``masked=False`` and ``axis`` is specified, then the
             output `~numpy.ndarray` will have the same shape as the
             input ``data`` and contain ``np.nan`` where values were
-            clipped. If the input ``data`` was a masked array, then the
-            output `~numpy.ndarray` will also contain ``np.nan`` where
-            the input mask was `True`. If ``return_bounds=True`` then
-            the returned minimum and maximum clipping thresholds will be
-            be `~numpy.ndarray`\\s.
+            clipped. In this case, integer-type ``data`` arrays will
+            be converted to `np.float32`. If the input ``data`` was a
```

Comment:
Here and below, you need `~numpy.float32` if you want to make a cross-package link (that's the reason for the readthedocs failure)

Reply:
Fixed!